### DISCUSSION TITLE
AllocateLike has a new UI and it is not working

### DISCUSSION THREAD

**Discussion by jellypuno** [zowe/zowe-explorer-vscode#1319] [Q&A]
Zowe Explorer v1.15.0 has updates for allocate like and I don't remember discussing it. It has  new UI and it is not working. Did we have an agreement to change the UI for allocate like?

**Comment by katelynienaber** [comment]
Hello, can someone else check this? I also cannot allocate like, but the CLI is receiving the correct new DS name & correct "like" DS name from ZE.

Also @jellypuno I am not sure what has changed for UI on allocate like, I think it is the same as before? We could add the name of the original data set as a value in the input box for the new DS's name...

**Reply by jellypuno** [reply]
That's weird. Earlier it is asking me for a model dataset that I would like to use to allocate like.

**Reply by jellypuno** [reply]
Now it is how I expected it 🤔

**Reply by jellypuno** [reply]
Is the attributes correct? Do you think it got affected by  the editing of attributes?

**Reply by katelynienaber** [reply]
Yes, I think if you try to do it through the command palette, it should ask you for this. Because you don't have one selected...

So, maybe it's a bug, though I'm not sure how you could call the function _without_ passing it a node, if you right-clicked on the node to Allocate Like P:

**Reply by katelynienaber** [reply]
It doesn't ask for attributes, it just asks for the name of the data set you want to copy from, and the name to give the new copy. Those were both correct when I checked but I'm still getting error 500. I will try with postman...
